site/public/js/configuration.js

```diff
@@ -3,13 +3,24 @@ $(document).ready(function() {
     updateEmailSeatingOption();
 
     function updateForumMessage() {
+        updateForumCategoryWarning();
```

Comment:
is `updateForumCategoryWarning()` used anywhere else? If not you should just do a
```
 $(#forum-category-warning).toggle();
 $(#forum-enabled-warning).toggle();

Reply:
Removed the function.